asm/fcr31.s

```diff
@@ -0,0 +1,69 @@
+.include "macros.inc"
+.section .text, "ax"
+.set noreorder
+.set noat
+
+glabel fcr_get_rounding_mode
+cfc1 $v0, $31
+jr $ra
+andi $v0, 0x3
```

Comment:
Only comment I have here is that these masks and flags have pretty names in PR/R4300.h with _LANGUAGE_ASSEMBLY defined (e.g. this 0x3 is `FPCSR_RM_MASK`)

Reply:
actually this might be the only one thats easy to drop in lol